index.bs

```diff
@@ -10722,6 +10660,64 @@ for the specific requirements that the use of
 </div>
 
 
+<h4 oldids="dom-unrestricteddouble dom-unrestrictedfloat idl-unrestricted-double idl-unrestricted-float"
+    id="Unrestricted" extended-attribute lt="Unrestricted">[Unrestricted]</h5>
+
+If the [{{Unrestricted}}] [=extended attribute=] appears on one of the [=float types=],
+it creates a new IDL type such that that when an ECMAScript Number is converted to the IDL type,
+IEEE 754 non-finite, and special "not a number" values (NaNs) are allowed. [[!IEEE-754]]
+
+The [{{Unrestricted}}]
+extended attribute must
+[=takes no arguments|take no arguments=].
+
+A type that is not a [=float type=] must not
+be [=extended attributes associated with|associated with=]
+the [{{Unrestricted}}] extended attribute.
+
+See the rules for converting ECMAScript values to the various IDL float types
+in [[#es-float]] and [[#es-double]]
+for the specific requirements that the use of
+[{{Unrestricted}}] entails.
+
+<div class="example">
+
+    In the following [=IDL fragment=],
+    two [=operations=] are declared that
+    take three {{double}} arguments; one uses
+    the [{{Unrestricted}}] [=extended attribute=]
+    on all three arguments, while the other does not:
+
+    <pre highlight="webidl">
+        [Exposed=Window]
+        interface GraphicsContext {
+          void setColorFloat(double red, double green, double blue);
+          void setColorUnrestrictedFloat([Unrestricted] double red, [Unrestricted] double green, [Unrestricted] double blue);
+        };
+    </pre>
+
+    In an ECMAScript implementation of the IDL, a call to setColor3f with
```

Comment:
What's `setColor3f`?

Reply:
A leftover from what I used to call the `double`‑taking `setColor` methods before I settled on `setColorDouble`.